addons/armory/XEH_postInit.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+// Talking Armory Boxes
+[QGVAR(Locker), "containerOpened", {
```

Comment:
Why not when Armory tablet is open instead?

Reply:
It's basically the same at this point, except I already did it for `containerOpened`

Reply:
It would make more sense to do it when Armory is opened, no?